README.md

```diff
@@ -24,50 +23,31 @@
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+### Clone repo
 
+### Open index.html on the browser
 
 
```

Comment:
- [ ] Keep in mind that some people checking your repo may not know what cloning a repo is nor how it's done. Try describing the process:
- Clone the repository using `git clone https://github.com/iammanny/My-Portfolio.git`
- Open the` index.html` file on your browser